android_app/app/src/main/java/com/health/openscale/core/bluetooth/BluetoothSenssun.java

```diff
@@ -115,7 +115,7 @@ private void parseMeasurement(byte[] data) {
         switch(data[5]) {
             case (byte)0xAA:
             case (byte)0xA0:
-                if (values > 1) {
+                if (weightStabilized) {
```

Comment:
You checked the weightStabilzed value before it will bet set?

Reply:
Not sure what you mean? The value is false until set to true [here](https://github.com/oliexdev/openScale/blob/7e644087266c866ee08d13f60f7c8f860e4a79d1/android_app/app/src/main/java/com/health/openscale/core/bluetooth/BluetoothSenssun.java#L126)

Reply:
What happened if the first parse measurement the weight stabilized is true? You have to call again parse measurement...

Reply:
It is set to true [here](https://github.com/oliexdev/openScale/blob/7e644087266c866ee08d13f60f7c8f860e4a79d1/android_app/app/src/main/java/com/health/openscale/core/bluetooth/BluetoothSenssun.java#L126) so if that happens, it will just go past the if condition, save the weight and weightStabilized and go on with the next step.

Reply:
Ok thanks I can't try this fix out but I guess you tested it.